### PR TITLE
add a doc for binary package

### PR DESCRIPTION
**Pull request by ran-huang**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Add a doc for binary package contents.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.2 (TiDB 6.2 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: binary-package.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+---
+title: TiDB 离线包
+summary: 了解 TiDB 离线包及其内容物。
+---
+
+# TiDB 离线包
```
Comment (TomShawn) [inline binary-package.md:6]: 可以在 [TiDB 工具下载 | PingCAP Docs](https://docs.pingcap.com/zh/tidb/dev/download-ecosystem-tools) 中也加一下本文档的链接吗？

File: binary-package.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+TiDB 提供两个离线包：`TiDB-community-server` 软件包和 `TiDB-community-toolkit` 软件包。下载地址见 [PingCAP 官网](https://pingcap.com/zh/product-community/)。
+
+`TiDB-community-server` 软件包包含以下内容物：
```
Comment (TomShawn) [inline binary-package.md:10]: 能否和 release manager 确认下，哪些版本的离线包会包含 PDF 版的用户文档？

Comment (ran-huang) [reply binary-package.md:10]: 社区版离线包里应该不包含 PDF 的

File: binary-package.md (hunk 1 above)
Comment (ran-huang) [reply binary-package.md:6]: Done

File: binary-package.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+| alertmanager-{version}-linux-amd64.tar.gz |  |
+| blackbox_exporter-{version}-linux-amd64.tar.gz |  |
+| prometheus-{version}-linux-amd64.tar.gz |  |
+| node_exporter-{version}-linux-amd64.tar.gz |  |
+| tiup-linux-amd64.tar.gz |  |
+| tiup-{version}-linux-amd64.tar.gz |  |
+| local_install.sh |  |
+| cluster-{version}-linux-amd64.tar.gz |  |
+| insight-{version}-linux-amd64.tar.gz |  |
+| diag-{version}-linux-amd64.tar.gz | 在 v6.0.0 新增 |
```
Comment (qiancai) [inline binary-package.md:29]:
```suggestion
| diag-{version}-linux-amd64.tar.gz | 从 v6.0.0 起新增 |
```

File: download-ecosystem-tools.md [hunk 4]
```diff
@@ -5,7 +5,7 @@ aliases: ['/docs-cn/dev/download-ecosystem-tools/','/docs-cn/dev/reference/tools
 
 # TiDB 工具下载
 
-本文介绍如何下载 TiDB 工具包以及 [TiUniManager](/tiunimanager/tiunimanager-overview.md)。
+本文介绍如何下载 TiDB 工具包以及 [TiUniManager](/tiunimanager/tiunimanager-overview.md)。关于 TiDB 工具包的内容，请查看 [TiDB 离线包](/binary-package.md)
```
Comment (TomShawn) [inline download-ecosystem-tools.md:8]:
```suggestion
本文介绍如何下载 TiDB 工具包以及 [TiUniManager](/tiunimanager/tiunimanager-overview.md)。关于 TiDB 工具包的内容，请查看 [TiDB 离线包](/binary-package.md)。
```

File: binary-package.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
+---
+title: TiDB 离线包
+summary: 了解 TiDB 离线包及其内容物。
+---
+
+# TiDB 离线包
+
```
Comment (TomShawn) [inline binary-package.md:7]:
```suggestion

在[使用 TiUP 离线部署 TiDB](/production-deployment-using-tiup.md#离线部署) 前，你需要在[官方下载页面](https://pingcap.com/zh/product#SelectProduct)选择对应版本的 TiDB server 离线镜像包（包含 TiUP 离线组件包）。

```

File: binary-package.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+TiDB 提供两个二进制离线包：`TiDB-community-server` 软件包和 `TiDB-community-toolkit` 软件包。下载地址见 [PingCAP 官网](https://pingcap.com/zh/product-community/)。
```
Comment (TomShawn) [inline binary-package.md:8]:
```suggestion
TiDB 提供两个二进制离线包：`TiDB-community-server` 软件包和 `TiDB-community-toolkit` 软件包。
```

**Review by TomShawn — APPROVED** [review]
rest LGTM

File: binary-package.md [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+
+# TiDB 离线包
+
+在[使用 TiUP 离线部署 TiDB](/production-deployment-using-tiup.md#离线部署) 前，你需要在[官方下载页面](https://pingcap.com/zh/product#SelectProduct)选择对应版本的 TiDB server 离线镜像包（包含 TiUP 离线组件包）。
+
+TiDB 提供两个二进制离线包：`TiDB-community-server` 软件包和 `TiDB-community-toolkit` 软件包。
+
+`TiDB-community-server` 软件包中包含以下内容：
+
+| 内容物 | 变更说明 |
```
Comment (qiancai) [inline binary-package.md:14]:
```suggestion
| 内容 | 变更说明 |
```

**Review by qiancai — APPROVED** [review]
Minor comments. Rest LGTM

**Pull request merged**